components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
so to me is bug fixing vs breaking peoples repository :) `getThumbnail() -> getThumbnail(size=(64, 64))` is fair but we have to be explicit and make it clear in release notes, docs, etc